### ISSUE TITLE
man_made=mine_shaft vs man_made=mineshaft

### ISSUE THREAD

**Issue by matkoniecz** [SomeoneElseOSM/SomeoneElse-style#21]
In this case `man_made=mine_shaft` is already replaced by `man_made=mineshaft`

See also https://wiki.openstreetmap.org/wiki/Tag:man_made%3Dmineshaft and https://wiki.openstreetmap.org/wiki/Tag:man_made%3Dmine_shaft and https://wiki.openstreetmap.org/wiki/Mechanical_Edits/Mateusz_Konieczny_-_bot_account/fix_many_obvious_typos which performed final replacement.

Sorry for not writing earlier, I have not considered that you would want notification if new value is already supported in the same way.

Would you prefer to get such issue when I start considering replacement of one tag by another as a clear alias?

Or only once bot edit is officially proposed?

Or only once bot edit replacing such tag with another is running?

Or only once tag is fully gone?

Would it be different if some tag is replaced only manually without proposing bot edit?

Would you prefer separate issue for each tag? Update existing open one with additional listings? Update existing one even if it is closed?

**Comment by SomeoneElseOSM** [comment]
> Would you prefer to get such issue when...

A message saying "I'm going to remove X tag on Y date" before it happens would be great.  Obviously "on Y date" might be something like "in the next month or so"; just so that when it does disappear I'm not scrabbling around looking for where it went.  I don't mind if you use one issue or several; as long as I get the message.

**Comment by matkoniecz** [comment]
OK, then I will create issues for ones where I have not notified you before tag went away and in future I will take steps to notify before it happens rather than after it happens

>  Obviously "on Y date" might be something like "in the next month or so";

I expect that it will be "no earlier than XYZ, likely within month from that date" - hopefully it is good enough

**Comment by SomeoneElseOSM** [comment]
One more thing just about mine shafts - tagging of those is a bit problematic because historic https://taginfo.openstreetmap.org/search?q=mine_shaft#values prefers one version and man_made https://taginfo.openstreetmap.org/search?q=mineshaft#values prefers another.  

As features are edited between historic and not, I'd expect that people won't always edit the underscore in the value, so without mapper education about this specific issue, I expect you're wasting your time trying to tidy these up.  For this reason I still [support both versions](https://taginfo.openstreetmap.org/tags/man_made=mine_shaft#projects).

**Comment by matkoniecz** [comment]
I guess that separate issue for https://github.com/SomeoneElseOSM/mkgmap_style_ajt is not needed?

> One more thing just about mine shafts - tagging of those is a bit problematic because historic https://taginfo.openstreetmap.org/search?q=mine_shaft#values prefers one version and man_made https://taginfo.openstreetmap.org/search?q=mineshaft#values prefers another.

Oh yes, I noticed. And getting them normalized on any one version would be likely opposed so I am not signing up for such fix.

**Comment by matkoniecz** [comment]
> For this reason I still [support both versions](https://taginfo.openstreetmap.org/tags/man_made=mine_shaft#projects).

And this, I guess, means that I should close this issue? Feel free to reopen if you plan to take some action here
